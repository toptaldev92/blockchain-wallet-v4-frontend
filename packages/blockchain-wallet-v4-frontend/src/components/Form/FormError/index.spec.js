import React from 'react'
import FormError from './index.js'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
jest.mock('blockchain-info-components', () => ({ Text: 'text' }))

describe('FormError', () => {
  it('renders correctly', () => {
    const component = renderer.create(<FormError />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
