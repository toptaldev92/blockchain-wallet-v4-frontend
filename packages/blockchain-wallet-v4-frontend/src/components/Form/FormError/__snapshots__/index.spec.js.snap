// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormError renders correctly 1`] = `
.c0 {
  position: relative;
}

<text
  className="c0"
  color="error"
  size="12px"
/>
`;
